flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/SqlCallSyntax.java

```diff
@@ -316,4 +316,23 @@ private String doUnParse(
                             CallSyntaxUtils.asSerializableOperand(operands.get(2)));
                 }
             };
+
+    SqlCallSyntax OVER =
+            ((sqlName, operands) -> {
+                String projection = operands.get(0).asSerializableString();
+                String order = operands.get(1).asSerializableString();
+                String rangeBounds =
+                        CallSyntaxUtils.overRangeToSerializableString(
+                                operands.get(2), operands.get(3));
+                if (operands.size() == 4) {
+                    return String.format("%s OVER(ORDER BY %s %s)", projection, order, rangeBounds);
+                } else {
+                    return String.format(
+                            "%s OVER(PARTITION BY %s ORDER BY %s %s)",
+                            projection,
+                            CallSyntaxUtils.asSerializableOperand(operands.get(4)),
+                            order,
+                            rangeBounds);
+                }
```

Comment:
Alright, then nevermind.
I just thought that Table API in batch-mode might support `OVER` w/o `ORDER BY` and that it would need to be covered here as well.

Reply:
Generally, good catch. I just think it's a broader issue to make `OVER` work properly in batch as well. I'll create a ticket for that.

Reply:
I created https://issues.apache.org/jira/browse/FLINK-33410